### DISCUSSION TITLE
Change Size of Icon (width & height)

### DISCUSSION THREAD

**Discussion by hairullana** [lucide-icons/lucide#948] [General]
How to change size of lucide icon?
I have been set css font-size on i tags but it's not working
They still give the size 24x24 px

**Comment by karsa-mistmere** [comment]
Font size won't directly affect icon sizes, at least that's not our default behaviour.

To size an icon, you need to set the `width` and `height` of the inline SVG elements once replaced by Lucide. These will generally have a `.lucide` CSS class but you can also add classes to your `<i>` elements for several custom sizes that will be transferred to the `<svg>` elements (inline styles won't be transferred I believe).

Then in CSS, you can easily define the widths and heights in em units for a font size relative sizing using the appropriate class selectors.

E.g.:
```html
<i icon-name="menu" class="lucide-relative"></i>
```

```css
.lucide-relative {
  width: 1em;
  height: 1em;
}
```

**Reply by hairullana** [reply]
> Font size won't directly affect icon sizes, at least that's not our default behaviour.
> 
> To size an icon, you need to set the `width` and `height` of the inline SVG elements once replaced by Lucide. These will generally have a `.lucide` CSS class but you can also add classes to your `<i>` elements for several custom sizes that will be transferred to the `<svg>` elements (inline styles won't be transferred I believe).
> 
> Then in CSS, you can easily define the widths and heights in em units for a font size relative sizing using the appropriate class selectors.
> 
> E.g.:
> 
> ```
> <i icon-name="menu" class="lucide-relative"></i>
> ```
> 
> ```css
> .lucide-relative {
>   width: 1em;
>   height: 1em;
> }
> ```

Thank you, that's work

**Reply by Zerotask** [reply]
Would be great to add a little explanation to documentation. I also stumbled about this

**Reply by karsa-mistmere** [reply]
The documentation is already being updated to include such information: https://github.com/lucide-icons/lucide/pull/1521
